### ISSUE TITLE
Using Astrometry without checkout.sh

### ISSUE THREAD

**Issue by bpartridge** [dstndstn/tractor#19]
I guess this is more an Astrometry bug, but since it's relevant to Tractor HEAD's use of Astrometry HEAD, I'm filing it here since Astrometry doesn't have an issue tracker. It shouldn't be necessary to create a local copy of Astrometry in a subdirectory the way [checkout.sh](https://github.com/dstndstn/tractor/blob/master/checkout.sh) does it. With a slight tweak, I was able to get Tractor (which I'm using as an installed package) to use the Astrometry package I installed through Homebrew according to the instructions [here using --HEAD](http://astrometry.net/doc/build.html#mac-os-x-using-homebrew). Specifically, I needed to add `util/ttime.py` to the list of files that Astrometry.net installs in its makefile; it just needs to be added to the line in `util/Makefile` that includes `yanny.py`. In Homebrew syntax that's:

```
inreplace 'util/Makefile', 'yanny.py', 'yanny.py ttime.py'
```

So in other words, Astrometry added a new file that Tractor uses, but it was never added to Astrometry's build system configuration, so it doesn't get installed when Astrometry is installed.

Not a high-priority issue, but useful to track as we add collaborators.
